### DISCUSSION TITLE
Ex Option for Line Numbers

### DISCUSSION THREAD

**Discussion by unrealapex** [kyx0r/nextvi#141] [Q&A] [answered]
It would be nice if there were Ex options for number and relative number.

**Comment by kyx0r** [comment, marked as answer]
How about using a global macro? For example:
```:&2\#```

The reason why such option doesn't exist is because it has no merit in Ex as it only affects vi.

**Reply by unrealapex** [reply]
This works. Thank you!

**Reply by kyx0r** [reply]
As of latest, # is no longer a special path expansion character, therefore the correct command is:
`:&2#`

Just so there are no surprises.

**Comment by kyx0r** [comment]
Hm, I suppose there are also options like hlp, which don't really work in ex mode (though they could). I guess it's more so because there is no vi keybind to activate it, or it's not used often enough to warrant one. So basically if there is already a vi keybind to activate something, there is no reason to add an ex option since everything is reachable from a global macro anyways.
